app/src/renderer/scripts/common.js

```diff
@@ -27,18 +37,6 @@ module.exports.ratToBigNumber = function(rat) {
   }
   return new BN(rat)
 }
-// TODO uncomment when validator comission is done
-// module.exports.parseValidatorShares = function(validator) {
```

Comment:
are these functions not usable now?

Reply:
they are decimals now and not `1000/2500000`